keyboards/handwired/108keyTrackpoint/rules.mk

```
@@ -0,0 +1,22 @@
+MCU = at90usb1286
+F_CPU = 16000000
+ARCH = AVR8
+F_USB = $(F_CPU)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE ?= no        # Console for debug(+400)
```

This section should use `=` throughout, rather than `?=`.

```suggestion
CONSOLE_ENABLE = no         # Console for debug(+400)
```